org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.java

```diff
@@ -1144,18 +1131,6 @@ public InitializeResult getInitializeResult() {
 
 	@Override
 	public void afterBuild(List<IResourceDescription.Delta> deltas) {
```

Comment:
We should remove the implements `IBuildListener` from the server, too.

Reply:
i wanted to keep it here so that you can simply kick in then you provide the new impl

Reply:
The final protocol is based on a regular request from the client. I don't the afterBuild will be used for that.

Reply:
ok. what about subclassing clients?

Reply:
Hmmm maybe mark it at least as deprecated?

Reply:
better so?

Reply:
I think so, yes.